### PR TITLE
Add script to verify bootnodes

### PR DESCRIPTION
**Pull request by zjb0807**
Closes: #1093

### CODE REVIEW

File: scripts/verify-bootnodes.sh [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+elif [ $1 = "acala" ]; then
+	IMAGE=acala/acala-node:1.0.1
+	CHAIN_NAME=acala
+	GENESIS=
+else
+	echo "not support $1"
+	exit 1
+fi
+
+docker rm -f bootnotes > /dev/null 2>&1 || true
```
Comment (xlc) [inline scripts/verify-bootnodes.sh:18]: why need this?

File: scripts/verify-bootnodes.sh [hunk 2]
```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ $1 = "karura" ]; then
+	IMAGE=acala/karura-node:1.0.1
```
Comment (xlc) [inline scripts/verify-bootnodes.sh:6]:
```suggestion
	IMAGE=acala/karura-node:latest
```

File: Makefile [hunk 3]
```diff
@@ -140,3 +140,7 @@ srtool-build-wasm-karura:
 .PHONY: generate-tokens
 generate-tokens:
 	./scripts/generate-tokens-and-predeploy-contracts.sh
+
+.PHONY: verify-bootnodes-karura
+verify-bootnodes-karura:
```
Comment (xlc) [inline Makefile:145]: I don't think we need this in makefile, developer don't need to run this

**Review by xlc — COMMENTED** [review]
can you also setup a Github action to run this every week?

File: scripts/verify-bootnodes.sh (hunk 1 above)
Comment (zjb0807) [reply scripts/verify-bootnodes.sh:18]: Deal with the situation of `ctrl+c` in local testing. 
Will not affect the process.

**Review by xlc — APPROVED** [review]

**Pull request merged**